feature/guia nuevas practicas (#1525)
mercadopago/devsite-docs: guides/best-practices/ux-for-checkouts/introduction.pt.md

# Boas práticas de UX para checkouts

> Para impulsionar seus negócios, criamos uma guía de boas práticas de UX para checkouts que vão te ajudar a melhorar a experiência de pagamento dos seus clientes, ter mais conversão e oferecer um serviço ainda melhor. 

Reviewer: ```suggestion
Para impulsionar seus negócios, criamos um guia de boas práticas de UX para checkouts que vai te ajudar a melhorar a experiência de pagamento dos seus clientes e aumentar as conversões de vendas.
```

![pt best practices introduction](/images/best-practices-guide/PortIntroduccionBuenasPracticas.png)

Neste guia, mostraremos como construir sua própria experiência de checkout através da implementação de nossas soluções, que permitem personalizar seu front-end e colocar essas dicas em prática.

> WARNING 
> 
> Importante
> 

> > Para saber cómo performa tu checkout una vez implementadas estas buenas prácticas, te recomendamos integrar alguna herramienta de trackeo y realizar optimizaciones en base a los resultados para mejorar la experiencia de tus usuarios y tener más conversiones.

Reviewer: @abderossi - Gosti, aqui está em ES.

> > Para saber cómo performa tu checkout una vez implementadas estas buenas prácticas, te recomendamos integrar alguna herramienta de trackeo y realizar optimizaciones en base a los resultados para mejorar la experiencia de tus usuarios y tener más conversiones.

Author: ```suggestion
> Para saber o desempenho do seu checkout após a implementação dessas boas práticas, recomendamos que você integre uma ferramenta de rastreamento e realize otimizações com base nos resultados para melhorar a experiência de seus usuários e ter mais conversões.
```

Author: @rwaetemann Listo! Cómo lo ves ahora?

Reviewer: Perfectooooo!

> NEXT_STEP_CARD_PT
>

> > Otimize sua experiência de checkout

Reviewer: ```suggestion
> Otimize a experiência de checkout
```
